app/controllers/TransitAccompanyingDocumentController.scala

```diff
@@ -33,7 +33,7 @@ import scala.xml.NodeSeq
 
 class TransitAccompanyingDocumentController @Inject()(
   conversionService: TransitAccompanyingDocumentConversionService,
-  pdf: TransitAccompanyingDocumentPdfGenerator,
+  pdf: UnloadingPermissionPdfGenerator,
```

Comment:
Is this name change correct? Does the controller need to change also?

Reply:
I'd actually reverted it back to UnloadingPermissionPdfGenerator when I noticed the xsds were the same. The UnloadingPermissionPdfGenerator and PermissionToStartUnloading need renaming because they will deal with both tad/up docs. The TransitAccompanyingDocumentController can stay, this controller will focus on an IE015 message and receiving an MRN (which is different to the other controller)